app/src/docs/v1.api-spec.yaml

```diff
@@ -1421,6 +1421,10 @@ paths:
     get:
       summary: Get a file
       operationId: fileGet
+      security:
+        - BasicAuth: []
+        - BearerAuth: []
+          OpenID: []
```

Comment:
Great! Can you also put in the HTTP 429 error? There are other examples in the file. Basically any endpoints that allow API Key access should have the rate limiting applied, and also document the 429 responses so that people building apps will _hopefully_ handle them!